src/lib/api/supabase/projects/logger.ts

```diff
@@ -2,65 +2,60 @@ import axios from 'axios'
 import { readNetwork } from 'constants/networks'
 import { getLogger } from 'lib/logger'
 
```

Comment:
This logger reports on updates to the DB Projects table. We prolly wanna deprecate this logger at some point, or at least not point it to discord. but for now just renaming for consistency

Reply:
@peripheralist we could nuke the discord right now, it's logging to Axiom here: https://github.com/jbx-protocol/juice-interface/pull/3334/files#diff-18d4ab6ed1a1e6ed9ff96d17dc78d2527a1c3b45b773908e5c8776d1dc1ca169R38